extensions-core/lookups-cached-global/src/main/java/org/apache/druid/query/lookup/NamespaceLookupIntrospectHandler.java

```diff
@@ -95,6 +95,13 @@ public Response getMap()
 
   private Map<String, String> getLatest()
   {
-    return ((MapLookupExtractor) factory.get()).getMap();
+    final LookupExtractor lookup = factory.get();
```

Comment:
should we just bake this into the `LookupExtractor` interface with a default impl that throws the unsupported exception?

Reply:
Yeah, that seems like a good idea. I removed `iterable` and `keySet` from the interface, and added `asMap`.